queries/soql/highlights.scm

```diff
@@ -0,0 +1,142 @@
+; https://code.visualstudio.com/api/language-extensions/semantic-highlight-guide#semantic-token-classification
```

Comment:
What's this comment here for?
```suggestion
```

Reply:
copied from the original scm, I will remove it